android/java/res/drawable/brave_rating_rate_now_bg.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@sujitacharya2005 you can run `npm run presubmit` to see if we have license header related issues.

Reply:
it would complain about few missing license headers

Reply:
resolved all presubmit error